backend/cmm.mli

```diff
@@ -104,6 +104,8 @@ type rec_flag = Nonrecursive | Recursive
 
 type prefetch_temporal_locality_hint = Nonlocal | Low | Moderate | High
 
+type atomic_op = Fetch_add | Fetch_sub | Fetch_and | Fetch_or | Fetch_xor | CAS
```

Comment:
I think it would be easier to maintain if the names here match the names of intrinsics in cmm_helpers, so I'd prefer `Fetch_and_add` and `Compare_and_swap`.

Reply:
Changed